pkg/asset/installconfig/basedomain.go

```diff
@@ -46,7 +46,15 @@ func (a *baseDomain) Generate(_ context.Context, parents asset.Parents) error {
 	case aws.Name:
 		a.BaseDomain, err = awsconfig.GetBaseDomain()
 		cause := errors.Cause(err)
-		if !(awsconfig.IsForbidden(cause) || request.IsErrorThrottle(cause)) {
+		defaultThrottles := []retry.IsErrorThrottle{retry.ThrottleErrorCode{Codes: retry.DefaultThrottleErrorCodes}}
+		isThrottleError := false
+		for _, df := range defaultThrottles {
+			isThrottleError = df.IsErrorThrottle(cause).Bool()
+			if isThrottleError {
+				break
+			}
+		}
+		if !(awsconfig.IsForbidden(cause) || isThrottleError) {
```

Comment:
Do we need to also migrate the `awsconfig.IsForbidden`, which still uses the V1 sdk? Or maybe a separate PR?